### DISCUSSION TITLE
Poor visual quality of wallpaper when QT_SCALE_FACTOR > 1

### DISCUSSION THREAD

**Discussion by denysformiga** [dat-linux/community#33] [Issues]
Hi,
I'm using DAT Linux 1.0.1-lxqt with a 4k monitor and noticed a low quality of wallpaper after change the QT_SCALE_FACTOR to be bigger than 1 to have menu bar, icons, etc. appropriate to my monitor. Looks good now, but I noticed that the image quality of the wallpaper becomes much poorer after the change. 

Before:
&nbsp;&nbsp;<img width='500' height='350' src="https://github.com/denysformiga/denysformiga/blob/main/bug1.jpg?raw=true" />&nbsp;&nbsp;

After:
  &nbsp;&nbsp;<img width='500' height='350' src="https://github.com/denysformiga/denysformiga/blob/main/bug2.jpg?raw=true" />&nbsp;&nbsp;&nbsp;

How can I fix that? 
Is anyone else facing the same problem?

**Comment by dat-linux** [comment]
Hi, I think the image is 1920p. Sorry there's not a larger one I can give you. You can change it to whatever image you want, just right click on the desktop -> Desktop Prefs -> Background. 

In general, I guess rescaling the desktop also rescales the bg image and this creates artefacts in the raster. I'm not sure if it's possible to avoid just the bg from rescaling, I would imagine not. You can try using a vector based image.

**Reply by denysformiga** [reply]
I'm using 4k images, and I saw the same problem in this discussion [here](https://github.com/lxqt/pcmanfm-qt/issues/1595).
Unfortunately, I don't understand how to apply the patch (12d0efb) mentioned in the “Possible Solution”. 

Can you help me?

**Reply by dat-linux** [reply]
Well, the fix was made in the pcmanfm-qt codebase, so you'd need to download and build it. The Lubuntu Jammy repos only have a previous stable version unfortunately, the newer version may be a part of 24.04 upgrade (which DAT Linux 2.0 will be based on). Note, installing a later version of pcmanfm could break stuff.

Alternatively, you could merge the patch from the latest branch back into the 0.17 branch, and then build/install that. If this is possible at all, as the patch may not be backwards compatible.

**Reply by denysformiga** [reply]
Thanks for explanation. Sadly, I'll live with that. Set some wallpaper with few details, maybe something like 8-bit, and move on.
